portfolio/src/main/webapp/slideshow.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Slideshow</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+<style>
+.mySlides {display: none;}
+img {vertical-align: middle;}
+
+/* Slideshow container */
+.slideshow-container {
+  max-width: 1000px;
+  max-height:700px;
+  position: relative;
+  margin: auto;
+}
+
+.active {
+  background-color: #717171;
+}
+
+/* Fading animation */
+.fade {
+  -webkit-animation-name: fade;
+  -webkit-animation-duration: 1.5s;
+  animation-name: fade;
+  animation-duration: 1.5s;
+}
+
+@-webkit-keyframes fade {
+  from {opacity: .4} 
+  to {opacity: 1}
+}
+
+@keyframes fade {
+  from {opacity: .4} 
+  to {opacity: 1}
+}
+
+/* On smaller screens, decrease text size */
+@media only screen and (max-width: 300px) {
+  .text {font-size: 11px}
+}
+body {margin:0;}
+
+  ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #333;
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+  }
+
+  li {
+    float: left;
+  }
+
+  li a {
+    display: block;
+    color: white;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+  }
+
+  li a:hover:not(.active) {
+    background-color: #111;
+  }
+
+  .active {
+    background-color: #4CAF50;
+  }
+</style>
+</head>
+<body>
+  <ul>
+    <li><a href="index.html">Home</a></li>
+    <li><a href="resume.html">Resume</a></li>
+    <li><a href="aboutme.html">About</a></li>
+    <li><a href="slideshow.html">Slideshow</a></li>
+    <li><a href="mailto:cjcoulibaly@gmail.com" target="_blank">Contact Me</a></li>
+  </ul>
+
+<div class="slideshow-container">
+
+<div class="mySlides fade">
+  <img src="images/noah1.jpg" style="width:100%">
+</div>
+
+<div class="mySlides fade">
+  <img src="images/fam1.jpg" style="width:50%">
+</div>
+
+<div class="mySlides fade">
+  <img src="images/mom1.jpg" style="width:60%">
+</div>
+
+<div class="mySlides fade">
+  <img src="images/noah2.jpg" style="width:100%">
+</div>
+
+<div class="mySlides fade">
+  <img src="images/fam2.jpg" style="width:60%">
+</div>
+
+<div class="mySlides fade">
+  <img src="images/brothers.jpg" style="width:80%">
+</div>
+</div>
+<br>
+
+<div style="text-align:center">
```

Comment:
Try removing all the "dot" related code, here and below in the script. 
If it stops working then maybe you have left some code that's still referring to the deleted element/variable (so the script stops working).